File: persiafava-php/examples/send_sms.php
<?php
require __DIR__ . '/../vendor/autoload.php';

use PersiaFava\Client;
use PersiaFava\Exceptions\ApiException;
use PersiaFava\Exceptions\HttpException;

$client = new Client('YOUR_API_KEY');

try {
    $result = $client->sendSms(['09123456789'], '3000569999', 'سلام! این یک پیام آزمایشی از SDK پرشیا فاواست.');
    print_r($result);
} catch (ApiException $e) {
    echo "خطای API: " . $e->getMessage();
} catch (HttpException $e) {
    echo "خطای شبکه: " . $e->getMessage();
}

File: persiafava-java/examples/SendSms.java
import com.persiafava.sms.Client;
import com.persiafava.sms.ApiException;
import com.persiafava.sms.HttpRequestException;

public class SendSms {
    public static void main(String[] args) {
        // روش اول: نام کاربری و رمز عبور
        Client client = new Client("YOUR_API_KEY");

        try {
            var result = client.sendSms(java.util.List.of("09123456789"), "3000569999", "سلام! این یک پیام آزمایشی از SDK پرشیا فاواست.");
            System.out.println(result);
        } catch (ApiException e) {
            System.out.println("خطای API: " + e.getMessage());
        } catch (HttpRequestException e) {
            System.out.println("خطای شبکه: " + e.getMessage());
        }
    }
}
